Azure/azure-cli
# [Containerapp] `az containerapp create/up`: Make `--target-port` optional

**Related command**
`az containerapp up --name port0verify2 --resource-group port0verify-rg --ingress external --image mcr.microsoft.com/azuredocs/containerapps-helloworld:latest`
`az containerapp create --name port0verify --resource-group WestCentralUsVerify --ingress external --image mcr.microsoft.com/azuredocs/containerapps-helloworld:latest --environment port0verify-env`

**Description**
The --target-port field is now optional so the validation needs to be updated accordingly.

**Testing Guide**
Before this change, creating an app without the --target-port field causes a warning `No ingress provided, defaulting to port 80. Try `az containerapp up --ingress external --target-port <port>` to set a custom port.`. Now that warning has been removed. Other commands may have reported the error `Usage error: must specify --target-port with --ingress` which has also been removed.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
